library/CMakeLists.txt

```diff
@@ -38,6 +38,20 @@ configure_file("${CMAKE_CURRENT_SOURCE_DIR}/include/hipsparse-version.h.in"
 configure_file("${CMAKE_CURRENT_SOURCE_DIR}/include/hipsparse.h"
                "${PROJECT_BINARY_DIR}/include/hipsparse/hipsparse.h" COPYONLY)
 
+set(HEADER_FILES 
+  level1/hipsparse_axpyi.h
```

Comment:
Do you think we should keep the same folder hierarchy from rocSPARSE? include/internal/level1?

Reply:
Sure ill do that.

Reply:
I still don't know if internal was really a good idea. I mimicked from blas.
but library/include/hipsparse might be a good idea.